runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/External.java

```diff
@@ -225,9 +251,25 @@ public OutputT expand(InputT input) {
         }
       }
 
+      ExpansionApi.ExpansionRequest.Builder requestBuilder =
+          ExpansionApi.ExpansionRequest.newBuilder();
+      if (!outputCoders.isEmpty()) {
```

Comment:
No need to gate on this--if it's empty it'll still do the right thing.

Reply:
Done.